test/functional/lua/vim_spec.lua

```diff
@@ -1326,7 +1326,9 @@ describe('lua stdlib', function()
     ]],
     }
     feed('<cr>')
-    eq({ 3, NIL }, api.nvim_get_var('yy'))
+    retry(nil, 200, function()
```

Comment:
I think the 200 should be the first argument since that is the number of tries. The second argument seems to be `max_ms`, which I assume means "rerun as much as possible until time runs out".

Reply:
Yes, that is what bfredl suggested.

Reply:
Well, it seems weird to me since the default value of `max_ms` is 10000 which is much bigger than 200. If it was intentional then it's fine.

Reply:
@bfredl ?

Reply:
Changed it to `10, nil` to match the retry of the test before; let's see if that works too.

Reply:
Seems to work as well, so let's go with that.